app/lib/fisbot/code_location_subscription.rb

```diff
@@ -1,6 +1,34 @@
 class CodeLocationSubscription < FisbotApi
-  def initialize(code_location_id)
-    @endpoint = 'subscriptions'
-    @data = { code_location_id: code_location_id }
+  class << self
+    def endpoint
+      :subscriptions
+    end
+
+    def code_locations_for_project(project_id)
+      uri = URI("#{api_uri}/api/v1/subscriptions/code_locations/#{project_id}.json?api_key=#{api_key}")
```

Comment:
We could move all these API related routes to some constant file and use some wrapper to just grep and replace, key with a value.

Reply:
We moved the routes to another class, greatly simplifying the url generation.